### DISCUSSION TITLE
Show error when subscribing to table that realtime isn't enabled on

### DISCUSSION THREAD

**Discussion by danhodkinson** [supabase/supabase#11622] [Feature Requests]
Would be great if when trying to subscribe to a table - if that table hasn't had realtime enabled then it provides some form of feedback to the user (developer). 
I was recently in full debug mode trying to figure out why i was receiving a payload for one subscription but not the other.

**Comment by GaryAustin1** [comment]
You can get this error now.  It is in the websocket messages...
![image](https://user-images.githubusercontent.com/54564956/212199644-dca2b5f8-77ca-48d1-b131-357a2eedf13c.png)

And you can get them in code with the extra .on at the bottom...
```
 const mySubscription = supabase1
        .channel('mychannel')
        .on('postgres_changes',
            { event: '*', schema: 'public', table: 'accounts'}, (payload) => {
                console.log('Change received!', payload)
            })
        .subscribe((status)=>{
            console.log('subscribe = status',status);
        })
       .on('system', {}, payload => {
        if (payload.extension === 'postgres_changes') {
            console.log('more info',payload);
        }
    })
 ```

![image](https://user-images.githubusercontent.com/54564956/212203492-81da635c-b780-4365-aebb-b7110c400017.png)

**Reply by danhodkinson** [reply]
Thanks @GaryAustin1 is this documented in the Realtime docs anywhere? I didn't spot the additional .on for system changes anywhere?

**Reply by GaryAustin1** [reply]
Not that I know of.  I got from Supabase realtime github issues.  It might also be in realtime-js github.

**Reply by w3b6x9** [reply]
@danhodkinson we have a PR open that specifies this that you can refer to: https://github.com/supabase/supabase/pull/10251.
